jib-core/src/main/java/com/google/cloud/tools/jib/docker/json/DockerLoadManifestTemplate.java

```diff
@@ -43,7 +43,7 @@
  * @see <a href="https://github.com/moby/moby/blob/master/image/tarexport/load.go">Docker load
  *     source</a>
  */
-public class DockerLoadManifestTemplate implements ListOfJsonTemplate {
+public class DockerLoadManifestTemplate implements ListOfJsonTemplate<JsonTemplate> {
```

Comment:
I think `ListOfJsonTemplate<DockerLoadManifestTemplate>` would be more applicable here.

Reply:
This sounds weird. The actual `DockerLoadManifestTemplate` class matches and represents a single Docker Manifest element. But `ListofJsonTemplate<itself>` also implies that the class can act as a list of Docker Manifests. This sounds confusing to me. I think I'll fix #997, and things should be significantly simplified.

Reply:
Sounds good. Let me know if that works out well - I feel like we had a reason we had to add the `ListOfJsonTemplate` class.